src/deadline/client/ui/widgets/host_requirements_tab.py

```diff
@@ -239,6 +277,23 @@ def get_requirements(self) -> List[Dict[str, Any]]:
             )
         return requirements
 
+    def set_requirements(self, requirements: List[Dict[str, Any]]):
+        for requirement in requirements:
+            if requirement.get("name") == "attr.worker.os.family":
+                if requirement.get("anyOf") not in self.OS_ROW_OPTIONS:
+                    raise ValueError(
+                        "Invalid value {} for option attr.worker.os.family".format(requirement.get("anyOf"))
+                    )
+                self.os_row.combo_box.setCurrentText(requirement.get("anyOf"))
+            if requirement.get("name") == "attr.worker.cpu.arch":
```

Comment:
I believe this should be `elif` instead of `if`? Otherwise the `else` condition will be taken if the requirement name is `"attr.worker.os.family"`?

Reply:
You are correct! Fixing now